assets/src/modules/Digitizing.js

```diff
@@ -1195,6 +1224,38 @@ export class Digitizing {
                         if (geojson) {
                             OL6features = (new GeoJSON()).readFeatures(geojson, options);
                         }
+                    } else if (fileExtension === 'fgb') {
+                        let features = [];
+
+                        const blob = new Blob([fileContent]);
+                        const stream = blob.stream();
+
+                        for await (const feature of flatgeobuf.ol.deserialize(stream, null, handleHeaderMeta)) {
+                            features.push(feature);
+                        }
+
+                        if (projFGB !== null) {
+                            const projCode = projFGB.split(":")[1];
+
+                            // We need a reprojection to be done because flatgeobuf files doessn't have a precise projection
+                            // We neither used wkt located in headers due to some errors to define it with proj4.
+                            // Nor 'fromEPSGcode()' because it returns a 'Projection' object that sometimes throws errors.
+                            let descriptor = await fetch('https://epsg.io/' + projCode + '.proj4');
+                            descriptor =  await descriptor.text();
+
+                            proj4.defs(projFGB, descriptor);
+                            register(proj4);
+
```

Comment:
Checking if the projCode is allready known before requesting epsg.io

Reply:
Should we check as well if the `descriptor` ended with a HTTP code < 400 ? (or similar)
https://fr.wikipedia.org/wiki/Liste_des_codes_HTTP

Reply:
> Should we check as well if the `descriptor` ended with a HTTP code < 400 ? (or similar) https://fr.wikipedia.org/wiki/Liste_des_codes_HTTP

I will replace my utilization of "fetch" by the one located directly in the [Utils module](https://github.com/3liz/lizmap-web-client/blob/master/assets/src/modules/Utils.js#L113) to take care of them.